tests/src/test/scala/com/evolutiongaming/kafka/journal/JournalIntSpec.scala

```diff
@@ -252,6 +252,9 @@ class JournalIntSpec extends AsyncWordSpec with JournalSuite {
       }
     }
   }
+
+  private def event(seqNr: SeqNr) =
+    Event[Payload](seqNr)
 }
 
 object JournalIntSpec {
```

Comment:
Would be also nice to have at least one integration test working with circe, what do you think?